### ISSUE TITLE
3.1.4 crash on Qt6Gui.dll QScreen::availableGeometry

### ISSUE THREAD

**Issue by nono303** [samsonwang/LaunchyQt#92]
**Describe the bug**
Unable to execute version 3.1.4
_version 3.1.2 work like a charm on the same system_

**To Reproduce**
1. download & unzip Launchy-3.1.4-win-x64.7z
2. run Launchy.exe
3. application always crash with a ~117MB dump
4. dump analysis report:
```
*******************************************************************************
*                                                                             *
*                        Exception Analysis                                   *
*                                                                             *
*******************************************************************************

*** WARNING: Unable to verify checksum for Launchy.exe

KEY_VALUES_STRING: 1

    Key  : AV.Dereference
    Value: NullClassPtr

    Key  : AV.Fault
    Value: Read

    Key  : Analysis.CPU.mSec
    Value: 484

    Key  : Analysis.DebugAnalysisManager
    Value: Create

    Key  : Analysis.Elapsed.mSec
    Value: 3925

    Key  : Analysis.IO.Other.Mb
    Value: 5

    Key  : Analysis.IO.Read.Mb
    Value: 0

    Key  : Analysis.IO.Write.Mb
    Value: 15

    Key  : Analysis.Init.CPU.mSec
    Value: 61

    Key  : Analysis.Init.Elapsed.mSec
    Value: 8560

    Key  : Analysis.Memory.CommitPeak.Mb
    Value: 91

    Key  : Timeline.OS.Boot.DeltaSec
    Value: 95697

    Key  : Timeline.Process.Start.DeltaSec
    Value: 1

    Key  : WER.OS.Branch
    Value: ni_release

    Key  : WER.OS.Timestamp
    Value: 2022-05-06T12:50:00Z

    Key  : WER.OS.Version
    Value: 10.0.22621.1

    Key  : WER.Process.Version
    Value: 3.1.3.0


FILE_IN_CAB:  Launchy.exe_230406_090944.dmp

COMMENT:  
*** "C:\bin\wscc_sysinternals\procdump.exe" -accepteula -ma -j "s:\telecharger" 28580 616 0000020B88490000
*** Just-In-Time debugger. PID: 28580 Event Handle: 616 JIT Context: .jdinfo 0x20b88490000

NTGLOBALFLAG:  0

APPLICATION_VERIFIER_FLAGS:  0

CONTEXT:  (.ecxr)
rax=0000000000000000 rbx=0000000000000048 rcx=0000000000000000
rdx=000000a5dacff6d0 rsi=0000020b83f512c0 rdi=000000a5dacff810
rip=00007ffea08221e0 rsp=000000a5dacff698 rbp=000000a5dacff719
 r8=000000a5dacff628  r9=0000000000000001 r10=0000000000000003
r11=000000a5dacff500 r12=0000000000000000 r13=0000000000000000
r14=0000020b860aa660 r15=0000020b83f512c0
iopl=0         nv up ei pl nz na po nc
cs=0033  ss=002b  ds=002b  es=002b  fs=0053  gs=002b             efl=00010206
Qt6Gui!QScreen::availableGeometry:
00007ffe`a08221e0 488b4108        mov     rax,qword ptr [rcx+8] ds:00000000`00000008=????????????????
Resetting default scope

EXCEPTION_RECORD:  (.exr -1)
ExceptionAddress: 00007ffea08221e0 (Qt6Gui!QScreen::availableGeometry)
   ExceptionCode: c0000005 (Access violation)
  ExceptionFlags: 00000000
NumberParameters: 2
   Parameter[0]: 0000000000000000
   Parameter[1]: 0000000000000008
Attempt to read from address 0000000000000008

PROCESS_NAME:  Launchy.exe

READ_ADDRESS:  0000000000000008 

ERROR_CODE: (NTSTATUS) 0xc0000005 - L'instruction   0x%p emploie l'adresse m moire 0x%p. L' tat de la m moire ne peut pas  tre %s.

EXCEPTION_CODE_STR:  c0000005

EXCEPTION_PARAMETER1:  0000000000000000

EXCEPTION_PARAMETER2:  0000000000000008

STACK_TEXT:  
000000a5`dacff698 00007ff6`736393ee     : 00000000`00000000 00000000`00000048 0000020b`83f512c0 000000a5`dacff810 : Qt6Gui!QScreen::availableGeometry
000000a5`dacff6a0 00007ff6`73635d6f     : 0000020b`83f4e4f0 0000020b`83f4e4f0 0000020b`83f4e4f0 00007ff6`73638730 : Launchy_exe+0x193ee
000000a5`dacff780 00007ff6`73662477     : 00000000`00000000 00000000`00000000 00000000`00000001 0000020b`83f512c0 : Launchy_exe+0x15d6f
000000a5`dacffa10 00007ff6`7365f431     : 00000000`00000000 0000020b`83f512c0 0000020b`83f25fd0 0000020b`83f0fb10 : Launchy_exe+0x42477
000000a5`dacffa40 00007ff6`73674c77     : 00000000`00000001 00000000`00000000 0000020b`00000000 00000000`00000000 : Launchy_exe+0x3f431
000000a5`dacffb70 00007ff6`73673cbe     : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 : Launchy_exe+0x54c77
000000a5`dacffc00 00007ffe`df9b269d     : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 : Launchy_exe+0x53cbe
000000a5`dacffc40 00007ffe`e0aea9f8     : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 : kernel32!BaseThreadInitThunk+0x1d
000000a5`dacffc70 00000000`00000000     : 00000000`00000000 00000000`00000000 00000000`00000000 00000000`00000000 : ntdll!RtlUserThreadStart+0x28


STACK_COMMAND:  ~0s; .ecxr ; kb

SYMBOL_NAME:  Qt6Gui!QScreen::availableGeometry+0

MODULE_NAME: Qt6Gui

IMAGE_NAME:  Qt6Gui.dll

FAILURE_BUCKET_ID:  NULL_CLASS_PTR_READ_c0000005_Qt6Gui.dll!QScreen::availableGeometry

OS_VERSION:  10.0.22621.1

BUILDLAB_STR:  ni_release

OSPLATFORM_TYPE:  x64

OSNAME:  Windows 10

IMAGE_VERSION:  6.2.4.0

FAILURE_ID_HASH:  {a7ea3afc-4681-51e1-c4ed-bd082676f2c6}

```

**Desktop (please complete the following information):**
 - Operation System: Win11 22h2 22621.1485
 - Launchy Version: 3.1.4 x64

**Comment by samsonwang** [comment]
bug fixed, nullptr related.

**Comment by nono303** [comment]
Thx @samsonwang!
Will you release it in next version or building a fixed 3.1.4 ?

**Comment by Cyberklabauter** [comment]
Thank you for fixing it. Great to see this project alive!
